src/hotspot/cpu/aarch64/macroAssembler_aarch64.cpp

```diff
@@ -1381,7 +1381,7 @@ void MacroAssembler::lookup_virtual_method(Register recv_klass,
   } else {
     vtable_offset_in_bytes += vtable_index.as_constant() * wordSize;
     ldr(method_result,
-        form_address(rscratch1, recv_klass, vtable_offset_in_bytes, 0));
+        form_address(rscratch1, recv_klass, vtable_offset_in_bytes, LogBytesPerWord));
```

Comment:
What is this change? Does not look equivalent to current code. Is this a bug fix?

Reply:
It's just something I noticed while scanning all of the calls to form_address. The shift should match the operand size.